manager/api/beans/src/main/java/io/apiman/manager/api/beans/summary/AvailableApiBean.java

```diff
@@ -35,10 +35,12 @@
     private String id;
     private String icon;
     private String endpoint;
+    private String routeEndpoint;

```

Comment:
I think the UI should probably add this as a link (when unmanaged) so the user can get to the service. An example would be http://myservice.vagrant.f8. When the service is imported we'd advice to remove the route, and it should then link to the service endpoint on the gateway if possible.